deploy/examples/monitoring/rbac.yaml

```diff
@@ -7,12 +7,32 @@ metadata:
   name: rook-ceph-monitor
   namespace: rook-ceph
 rules:
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - '*'
-  verbs:
-  - '*'
+  - apiGroups:
+      - monitoring.coreos.com
+    resources:
+      - cephclients
```

Comment:
We don't need all of the rook CRDs in this list, they don't apply to the monitoring.coreos.com api group. I believe we would only need `servicemonitors` in this list. Another possibility is `promeusrules`, but the rook operator doesn't create those anymore so it shouldn't need access to that.

Reply:
Yaa that's right,
I just didn't saw that its for monitor

Thanks